fix T1327096 (#268)
DevExpress/dotnet-eud: eud-for-devexpress-reports/reporting-for-web/report-designer/create-reports/master-detail-reports-with-detail-report-bands.md

    ![](../../images/eurd-web-master-detail-drop-fields-for-detail-layout.png)

>     Drag fields from the data category that matches the current detail level (the table specified in the most inner report’s DataMember). This ensures that each detail record is displayed correctly.

Reviewer: ```suggestion
    Drag fields from the data category that matches the current detail level, that is, the table specified in the innermost report’s DataMember. This ensures that each detail record is displayed correctly.
```

>     You can display parent values in a detail row using expressions (for example, [Categories.CategoryName]). If a field is not found at the current level, the expression engine automatically resolves it from the parent level.

Reviewer: ```suggestion
    You can display parent values in a detail row using expressions, for example, [Categories.CategoryName]. If a field is not found at the current level, the expression engine automatically resolves it from the parent level.
```

    However:

>     - If multiple levels contain the same field path, the result may be ambiguous and a value from an unexpected level can be used.

Reviewer: ```suggestion
    - If multiple levels contain the same field path, the result may be ambiguous, and a value from an unintended level may be used.
```

>     - If you use a field from another table within the same data source, the first record from that table will be shown for all rows.

Reviewer: ```suggestion
    - If you use a field from another table within the same data source, the first record from that table is shown for all rows.
```

>     - If you use a field from a different data source, no data will be displayed.

Reviewer: ```suggestion
    - If you use a field from a different data source, no data is displayed.
```
